recipes/faunus/meta.yaml

```diff
@@ -0,0 +1,68 @@
+{% set name = "faunus" %}
+{% set version = "2.1.1" %}
+{% set commit = "afba113c6008139fab704e682305847b5ae1d872" %}
+{% set sha256 = "8a224f8be892b06026de21e85d5c42ab30f81b9044917bdc87afcc81834c7044" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  # uncommented while staged
+  # fn: {{ version }}.tar.gz
+  # url: https://github.com/mlund/{{ name }}/archive/v{{ version }}.tar.gz
+  fn: {{ commit }}.tar.gz
+  url: https://github.com/mlund/{{ name }}/archive/{{ commit }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: true  # [win]
+  script:
+    - unset _CONDA_PYTHON_SYSCONFIGDATA_NAME  # [linux]
+    - cmake -DENABLE_MPI=on -DVERSION_STRING=v{{ version }} -DCMAKE_INSTALL_PREFIX:PATH=$PREFIX
```

Comment:
You have `-DENABLE_MPI=on` here for all builds, but you are still building a nompi variant. Do you want to build a version without mpi? It's okay for a package to only have mpi builds if that makes more sense.

Reply:
Thank, Min, I'm checking the mpi issue and it appears to be an related to command line arguments and the docopt.cpp package we're using. Merely linking with certain version of mpich/openmpi causes the segmentation fault.